blog/src/data/blog-posts.js

```diff
@@ -0,0 +1,246 @@
+        authorImage: images.author8,
+        date: "10 June",
+        image: images.blogImg18,
+        content: "Insights into how artists can thrive as entrepreneurs while staying true to their creative vision.",
+    },
+    {
+        id: 19,
+        title: "Digital Privacy in the 21st Century: What You Need to Know",
+        author: "Mia Anderson",
+        authorImage: images.author9,
+        date: "15 July",
+        image: images.blogImg19,
+        content: "Understanding the importance of digital privacy and tips to protect your online presence.",
+    },
+    {
+        id: 20,
+        title: "Beyond the Screen: The Power of Offline Connections",
+        author: "Noah Patel",
+        authorImage: images.author10,
+        date: "20 August",
+        image: images.blogImg20,
+        content: "Exploring the value of in-person interactions in an increasingly digital world.",
+    },
+    {
+        id: 21,
+        title: "The Evolving Landscape of Social Media",
+        author: "Emma Johnson",
+        authorImage: images.author1,
+        date: "10 September",
+        image: images.blogImg21,
+        content: "Analyzing the changing trends and impact of social media platforms on society.",
+    },
+    {
+        id: 22,
+        title: "Unlocking Creativity: Embracing Vulnerability in Art",
+        author: "Daniel Carter",
+        authorImage: images.author2,
+        date: "05 October",
+        image: images.blogImg22,
+        content: "How embracing vulnerability can enhance artistic expression and creativity.",
+    },
+    {
+        id: 23,
+        title: "The Psychology of Productivity: Finding Your Flow",
+        author: "Sophia Patel",
+        authorImage: images.author3,
+        date: "15 November",
+        image: images.blogImg23,
+        content: "Exploring the psychological factors that contribute to optimal productivity and focus.",
+    },
+    {
+        id: 24,
+        title: "Culinary Adventures: Exploring Global Flavors",
+        author: "Liam Turner",
+        authorImage: images.author4,
+        date: "20 December",
+        image: images.blogImg24,
+        content: "Embark on a gastronomic journey by sampling diverse cuisines from around the world.",
+    }
+];
+
+export const linksData = [
+    {
+        id: "read-links",
+        title: "Read Our",
+        links: ["Journals", "Articles","Biograpahy", "Old records","Audio version"]
+    },
+    {
+        id: "products-links",
+        title: "Products",
+        links: ["Reading collection", "Library", "Broadcast", "Podcast", "Book Series"]
+    },
+    
+    {
+        id: "quick-links",
+        title: "Quick Links",
+        links: ["Blogs", "Featured", "Latest", "Illustrations", "Latest"]
+    },
+    {
+        id: "more-links",
+        title: "More",
+        links: ["License","About Us", "Our policy"]
+    }
+]
+
+export const socialLinks = ["Twitter", "Linkedin" ,"Instagram" ,"Google" ,"Youtube"];
```

Comment:
I think creating object for social links would be best so that we can have title and URL of the social app. 
For Example:
```js
{title: "Twitter", URL: "abc.com"}
```